.github/workflows/fleather.yml

```diff
@@ -1,10 +1,6 @@
 name: build
 
-on:
-  push:
-    branches:
-      - 'master'
-  pull_request:
+on: push
```

Comment:
I found this https://github.com/orgs/community/discussions/26276#discussioncomment-3251140
Isn't what we want?

Reply:
I thought you wanted to have the CI run for other branches without a PR? What you've suggested wont do that.
Is there a problem with what we have now? It runs workflow for every push on any branches.

https://github.com/fleather-editor/fleather/pull/384#discussion_r1701325530

Reply:
No I think we should run at least on master and PRs
I'm fine with running it on branches and PRs

Reply:
I think what I did solves the duplicated run and it would also run for other branches. @amantoux

Reply:
As discussed, we need to do some more research